### ISSUE TITLE
Enhancement: Option to change how text enters window

### ISSUE THREAD

**Issue by MCas86** [3dproger/AxelChat#391]
Enhancement: Setting option to change how text enters window
- left
- right
- bottom
- just appear

the animation of it flying in from the right is distracting to some

**Comment by Aeon1704** [comment]
Please yes! And for Widget too.

**Comment by 3dproger** [comment]
Good idea, I'll try to do it after the advanced customization of appearance is ready

**Comment by synthodome** [comment]
Bumping this as I have a similar issue with the browser widget- New messages start at the top when I prefer new messages to show at the bottom.

<img width="583" height="388" alt="Image" src="https://github.com/user-attachments/assets/0f5ac4f5-9790-4845-9a63-07358ee2123d" />

As far as I'm aware there isn't an option to show new messages at the bottom of the widget, and the workaround is to spam messages until new messages appear at the bottom. It would be nice to have the option for messages to start at the bottom and scroll up.

Edit: resolved by @Aeon1704 

> ```
> .messagesListView {
> width: 100%;
> position: fixed;
> bottom: 0;
> display: flex;
> flex-direction: column;
> }
> ```
> 
> should help new messages to appear at bottom. You need to right click on your OBS Widget Element - Properties. In opened window should be CSS field. Add any of those codes ( do not replace text which already there), and it should take effect. As alternative to AxelChat lack of such settings, you can use Window-Preview of your OBS Widget as a stylized multi chat window.

**Comment by Aeon1704** [comment]
> Bumping this as I have a similar issue with the browser widget- New messages start at the top when I prefer new messages to show at the bottom.

There is a solution to his. I didnt try to change the order of how message appears(i.e. timestamp nickname etc) but this:
```
.messagesListView{
width: 100%;
position: fixed;
top: 0;
  display:flex;
  flex-direction: column-reverse;
}
```

will help your messages to appear at top. While this:
```
.messagesListView{
width: 100%;
position: fixed;
bottom: 0;
  display:flex;
  flex-direction: column;
}
```
should help new messages to appear at bottom. You need to right click on your OBS Widget Element - Properties. In opened window should be CSS field. Add any of those codes ( do not replace text which already there), and it should take effect. As alternative to AxelChat lack of such settings, you can use Window-Preview of your OBS Widget as a stylized multi chat window.

**Comment by synthodome** [comment]
@Aeon1704 yes that solution works and I marked my comment as resolved.
